hazelcast/src/test/java/com/hazelcast/map/MapIndexLifecycleTest.java

```diff
@@ -71,6 +72,7 @@
 
 @RunWith(HazelcastParallelClassRunner.class)
 @Category({QuickTest.class, ParallelTest.class})
+@Repeat(1000)
```

Comment:
leftover ?

Reply:
I think that's the reason for the `[DO NOT MERGE]`. Although it may crash our PR builder runtime statistics :p

Reply:
Didn't see that, time to sleep then :)

Reply:
Yes, it was intended to run the test several times in the PR builder in parallel with other tests. @tkountis thanks for the review and hope you had a good rest ;-)

PR is now updated without `@Repeat` and reasoning in description / see also https://github.com/hazelcast/hazelcast/issues/12597#issuecomment-373324317